snuba/admin/static/cardinality_analyzer/query_display.tsx

```diff
@@ -25,7 +27,7 @@ function QueryDisplay(props: {
   const [query, setQuery] = useState<QueryState>({});
   const [queryResultHistory, setCardinalityQueryResultHistory] = useState<
     CardinalityQueryResult[]
-  >([]);
+  >(getRecentHistory(HISTORY_KEY));
```

Comment:
Nit: I wonder if we could use `window.location.href` instead of defining different HISTORY_KEY in different files. I see that the href is different on different pages

```
On tracing tag
console.log(window.location.href)
https://snuba-admin.getsentry.net/#tracing

On system queries tab
console.log(window.location.href)
https://snuba-admin.getsentry.net/#system-queries

On cardinality analyzer tab
console.log(window.location.href)
https://snuba-admin.getsentry.net/#cardinality-analyzer
```

Reply:
I think this would cause really unexpected behaviour if we ever changed the urls (suddenly losing history). And the current behaviour of defining a key means you can be explicit about it and reuse results between pages if there's ever a use-case for that